### PR TITLE
display only the last lines in Log

### PR DESCRIPTION
**Pull request by volkerrichert**
Ok, clear log may not be needed anymore

### CODE REVIEW

**Comment by pazaan** [comment]
Thanks for the PR! There's a few things changing in here at the same time, so I need a bit more time to cherry-pick the merge. @volkerrichert are you on Gitter? I think more direct chat would be helpful.

**Comment by volkerrichert** [comment]
Hi pazaan,

not yet registered. Just reading. But I can join

File: app/src/main/java/info/nightscout/android/medtronic/MainActivity.java [hunk 1]
```diff
@@ -575,6 +628,8 @@ public void onReceive(Context context, Intent intent) {
 
             long nextPoll = pumpStatusData.getEventDate().getTime() + MedtronicCnlIntentService.POLL_GRACE_PERIOD_MS + MedtronicCnlIntentService.POLL_PERIOD_MS;
             startCgmService(nextPoll);
+            Log.d(TAG, "Lokal time " + new Date());
```
Comment (pazaan) [inline app/src/main/java/info/nightscout/android/medtronic/MainActivity.java]: Lokal = local

File: app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java [hunk 2]
```diff
@@ -43,9 +43,9 @@
 /**
  * Created by lgoedhart on 24/03/2016.
  */
-public class MedtronicCnlReader implements ContourNextLinkMessageHandler {
+public class MedtronicCNLReader implements ContourNextLinkMessageHandler {
```
Comment (pazaan) [inline app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java]: Change back to MedtronicCnlReader

File: app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java [hunk 3]
```diff
@@ -119,7 +119,7 @@ public MedtronicCNLSession getPumpSession() {
         } while (bytesRead > 0 && messageSize == 60);
 
         String responseString = HexDump.dumpHexString(responseMessage.toByteArray());
-        Log.d(TAG, "READ: " + responseString);
+        Log.d(TAG, "READ (" + this.getClass().getSimpleName() + "): " + responseString);
```
Comment (pazaan) [inline app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java]: Doesn't Log.d() already put the context in the log?

File: app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlIntentService.java [hunk 4]
```diff
@@ -237,12 +240,12 @@ protected void onHandleIntent(Intent intent) {
                         realm.cancelTransaction();
                     }
                 }
-
-                cnlReader.closeConnection();
             } catch (UnexpectedMessageException e) {
                 Log.e(TAG, "Unexpected Message", e);
                 sendStatus("Communication Error: " + e.getMessage());
             } finally {
+                // FIXME: 05.11.2016 has the close to be here?
```
Comment (pazaan) [inline app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlIntentService.java]: If this close() has been tested, the FIXME comment can be removed :)

File: app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java (hunk 2 above)
Comment (volkerrichert) [inline app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java]: But the file is called MedtronicCNLReader.java. The name should be thw same

File: app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java (hunk 3 above)
Comment (volkerrichert) [inline app/src/main/java/info/nightscout/android/medtronic/MedtronicCNLReader.java]: i have to check this. But there was a reason to print it.

File: app/src/main/java/info/nightscout/android/medtronic/MainActivity.java (hunk 1 above)
Comment (volkerrichert) [inline app/src/main/java/info/nightscout/android/medtronic/MainActivity.java]: :)

**Pull request merged**